### ISSUE TITLE
Removing all items doesn't restore the initial appearance

### ISSUE THREAD

**Issue by strohhut** [FineUploader/fine-uploader#1649]
## Type of issue
bug

## Uploader type 
all

### Fine Uploader version
5.11.5

### Browsers where the bug is reproducible
all

### Operating systems where the bug is reproducible
all

### Exact steps required to reproduce the issue
1. On the demo page under "Manually Trigger Uploads & Edit File Names" click "Select files" and select a file.
2. Cancel the file.

### All of your Fine Uploader initialization JavaScript code
The one from the demo page for example.

### Your Fine Uploader template markup (if using Fine Uploader UI and the issue is UI-related)
The one from the demo page for example.

### Detailed explanation of the problem 
I think it's a bit confusing that there are two different visual representations of the "nothing is selected state". A user might remove all items and then expects this to be like he never selected any items at all (at least when the items were only selected and not uploaded (`autoUpload = false`)). But the visual feedback suggests that something is different now:

![2](https://cloud.githubusercontent.com/assets/15095059/17804566/7f9587fa-65fc-11e6-95d4-93db0efe1dc6.JPG)

![1](https://cloud.githubusercontent.com/assets/15095059/17804523/4e91a5e4-65fc-11e6-9a85-2a5c194bf9e4.JPG)

**Comment by rnicholus** [comment]
The drop zone message is meant to be informative and instructional. After the user has already selected files, my assumption was that they are already familiar with the procedure and instructions are no longer necessary. But, I can see your point. I'll look into ensuring that a lack of files always re-renders the drop zone text.

**Comment by strohhut** [comment]
Yes, it's mainly a psychologial thing (missing the _satisfaction_ that the initial look is restored after realizing that selecting any files was a mistake and canceling them.) When I acted as test user the problem wasn't (as you already said) that I feared that I can't drop files any longer.

Unfortunately I'm a beginner at fine uploader and I'm no JavaScript programmer but I guess reverting `uploaderEl.removeAttribute(DROPZPONE_TEXT_ATTR);` in the cancel event handler when the number of files is zero should be a start.

**Comment by rnicholus** [comment]
Unfortunately, there are no small changes with a library of this age and complexity. Manual workflow and regression testing is needed for all changes, and changes to the drop code are especially concerning. I will keep this bug in mind, but development has mostly been a one person job for the last four years, so i have to prioritize, especially since this is more of a hobby than anything else. 

Are you a designer, or...?

**Comment by strohhut** [comment]
I fully understand. I regard this issue as having very low priority too. :)

I'm mainly a C# dev. Sometimes I'm hacking some TypeScript though. At first I tried Dropzone.js because there are TypeScript type definitions for it on DefinitelyTyped but now I'm using fine-uploader because I think the programming model is easier to use.

**Comment by rnicholus** [comment]
Hmm type definitions, that world be a good thing to add I guess

**Comment by strohhut** [comment]
If I ever gonna need more than the basic fine-uploader options and if I find the time I'll definitely consider making a definition file. :)

**Comment by rnicholus** [comment]
I plan on creating an issue to track this. Perhaps someone (maybe even myself) will pick this up when the need arises.
